core/src/main/scala/zio/logging/Logging.scala

```diff
@@ -27,32 +20,29 @@ object Logging {
   def error(cause: Cause[Any]): ZIO[Logging, Nothing, Unit] =
     log(LogLevel.Error)(cause.prettyPrint)
 
-  val ignore: UIO[Logging] =
+  val ignore: ZLayer[Any, Nothing, Logging] =
     make((_, _) => ZIO.unit)
 
   def locally[R1 <: Logging, E, A1](f: LogContext => LogContext)(zio: ZIO[R1, E, A1]): ZIO[R1, E, A1] =
-    ZIO.accessM[R1](_.logger.locally(f)(zio))
+    ZIO.accessM[R1](_.get.locally(f)(zio))
```

Comment:
I think you can actually omit the type annotation here and below since it can be inferred from the expected return type of the method.

Reply:
I will try